tests/components/vesync/snapshots/test_diagnostics.ambr

```diff
@@ -0,0 +1,274 @@
+    }),
+    'device_image': '',
+    'device_name': 'Fan',
+    'device_status': 'unknown',
+    'device_type': 'LV-PUR131S',
+    'extension': None,
+    'home_assistant': dict({
+      'disabled': False,
+      'disabled_by': None,
+      'entities': list([
+        dict({
+          'device_class': None,
+          'disabled': False,
+          'disabled_by': None,
+          'domain': 'fan',
+          'entity_category': None,
+          'entity_id': 'fan.fan',
+          'icon': None,
+          'name': None,
+          'original_device_class': None,
+          'original_icon': None,
+          'original_name': 'Fan',
+          'state': dict({
+            'attributes': dict({
+              'friendly_name': 'Fan',
+              'preset_modes': list([
+                'auto',
+                'sleep',
+              ]),
+              'supported_features': 1,
+            }),
+            'entity_id': 'fan.fan',
+            'last_changed': str,
+            'last_updated': str,
+            'state': 'unavailable',
+          }),
+          'unit_of_measurement': None,
+        }),
+        dict({
+          'device_class': None,
+          'disabled': False,
+          'disabled_by': None,
+          'domain': 'sensor',
+          'entity_category': 'diagnostic',
+          'entity_id': 'sensor.fan_filter_life',
+          'icon': None,
+          'name': None,
+          'original_device_class': None,
+          'original_icon': None,
+          'original_name': 'Fan Filter Life',
+          'state': dict({
+            'attributes': dict({
+              'friendly_name': 'Fan Filter Life',
+              'state_class': 'measurement',
+              'unit_of_measurement': '%',
+            }),
+            'entity_id': 'sensor.fan_filter_life',
+            'last_changed': str,
+            'last_updated': str,
+            'state': 'unavailable',
+          }),
+          'unit_of_measurement': '%',
+        }),
+        dict({
+          'device_class': None,
+          'disabled': False,
+          'disabled_by': None,
+          'domain': 'sensor',
+          'entity_category': None,
+          'entity_id': 'sensor.fan_air_quality',
+          'icon': None,
+          'name': None,
+          'original_device_class': None,
+          'original_icon': None,
+          'original_name': 'Fan Air Quality',
+          'state': dict({
+            'attributes': dict({
+              'friendly_name': 'Fan Air Quality',
+            }),
+            'entity_id': 'sensor.fan_air_quality',
+            'last_changed': str,
+            'last_updated': str,
+            'state': 'unavailable',
+          }),
+          'unit_of_measurement': None,
+        }),
+      ]),
+      'name': 'Fan',
+      'name_by_user': None,
+    }),
+    'mac_id': '00:00:00:00:00:00',
+    'manager': '**REDACTED**',
+    'mode': None,
+    'speed': None,
+    'sub_device_no': None,
+    'type': 'wifi-air',
+    'uuid': '00000000-1111-2222-3333-444444444444',
```

Comment:
should this be redacted?

Reply:
@balloob, are there any guidelines for what should be redacted?  You've highlighted uuid, mac_id, and account_id.  I don't know how uuid and mac_id are used, so I have no problem redacting them.  However, I made a conscious decision to leave account_id visible to allow the user to validate that they were using the correct account.  This could have been faulty logic on my part.

Reply:
redacted